src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClientX/IO/TdsStream.cs

```diff
@@ -147,31 +152,19 @@ public override void Write(byte[] buffer, int offset, int count)
         /// <param name="ct"></param>
         /// <returns></returns>
         /// <exception cref="NotImplementedException"></exception>
-        public override async Task FlushAsync(CancellationToken ct)
-            => await _writeStream.FlushAsync(ct).ConfigureAwait(false);
```

Comment:
Is this intentional? `TdsWriteStream.FlushAsync` is implemented.

Reply:
Sorry my bad. I will fix it .

Reply:
Also @edwardneal  Thanks for catching it. Appreciate the thorough review.